api4/user.go

```diff
@@ -42,6 +43,8 @@ func InitUser() {
 	BaseRoutes.User.Handle("/sessions", ApiSessionRequired(getSessions)).Methods("GET")
 	BaseRoutes.User.Handle("/sessions/revoke", ApiSessionRequired(revokeSession)).Methods("POST")
 	BaseRoutes.User.Handle("/audits", ApiSessionRequired(getAudits)).Methods("GET")
+
+	BaseRoutes.Users.Handle("/autocomplete/", ApiSessionRequired(autocompleteUsers)).Methods("GET")
```

Comment:
Can you remove the `/` at the end so that it's just `/autocomplete`, like the rest of the endpoints?